### PR TITLE
fix defaultFloat values

### LINKED ISSUE #456: using floats in @nearBindgen classes causes compilation error
**Describe the bug**

Whenever a `@nearBindgen` class has a `f32` or `f64` property without a default value causes a compilation error.

Error

**To Reproduce**

```ts
@nearBindgen
class Foo {
  someFloat: f32
}
```

**Expected behavior**

Use zero as the default value if none is provided (similar to `u64` for example).

**Screenshots**

Some Code

**Desktop (please complete the following information):**
 - OS: Linux

### PR DESCRIPTION
**Pull request by mehtaphysical**
Fixes #456

### CODE REVIEW

File: bindgen/assembly/index.ts [hunk 1]
```diff
@@ -484,7 +484,7 @@ function decode<T, V = Uint8Array>(buf: V, name: string = ""): T {
 // @ts-ignore
 @global
 function defaultValue<T>(): T {
-  if (isInteger<T>()) {
+  if (isInteger<T>() || isFloat<T>()) {
```
Comment (mehtaphysical) [inline bindgen/assembly/index.ts:487]: Would like to use `isNumber<T>()` here instead, but it doesn't work for some reason.

**Review by willemneal — APPROVED** [review]
Thanks!

**Comment by mehtaphysical** [comment]
I don’t have write access btw so I can’t merge.

**Comment by willemneal** [comment]
@mehtaphysical Thanks again, just wanted to test locally before merge.  BTW in the future could you name the branch you're working on so as to not overload `master`.

**Comment by mehtaphysical** [comment]
@willemneal Thanks!

Yes for sure. Thanks for the feedback about the branch naming.

**Pull request merged**